pkg/chain/ethereum/gen/contract/Deposit.go

```diff
@@ -96,12 +114,12 @@ func NewDeposit(
 // ----- Non-const Methods ------
 
 // Transaction submission.
-func (d *Deposit) NotifyRedemptionProofTimedOut(
+func (d *Deposit) NotifyCourtesyCall(
```

Comment:
I've navigated to pkg/chain/ethereum/gen and `make clean`. Then in /solidity `npm i`and finally `go generate ./...` `NotifyCourtesyCall` was actually named `ProvideRedemptionSignature`. How do we chose the name?

Reply:
ah, nvm, it's not sync generated

Reply:
Those functions are generated in a random order, this was fixed in https://github.com/keep-network/keep-common/pull/68 but the cutoff commit from where we branched off this fix does not include the referenced change.